cluster/sbin/snappy-locator.sh

```diff
@@ -37,12 +38,43 @@ shift
 . "$SNAPPY_HOME/bin/load-spark-env.sh"
 . "$SNAPPY_HOME/bin/load-snappy-env.sh"
 
-
+noOfInputsArgs=$#
 
 # Start up  the locator instance
 function start_instance {
-  "$SNAPPY_HOME"/bin/snappy locator "$mode" "$@"
+ "$SNAPPY_HOME"/bin/snappy locator "$mode" "$@"
 }
 
+if [ $noOfInputsArgs -le 1 ]
```

Comment:
Same concern as above.

Reply:
Resolved: Modified the code, now as per the current modification, if the user doesn't provide -dir argument, ask the user to provide it.